cat/index.adoc

```diff
@@ -4,7 +4,17 @@
 
 image::/images/jbosstools-cat-logo.png[JBoss Tools CAT Logo]
 
-Welcome to JBoss Tools Community Acceptance Testing (JBoss Tools CAT)!
+*Notice - As of August 1, 2016, the JBoss Tools Community Acceptance Testing (JBoss Tools CAT) will suspend
+operations. Many thanks to everyone who participated!*
+
+In place of the JBoss Tools CAT, we'd like to ask everyone who would
+like to contribute issues to JBoss Tools to enable the Eclipse Automated
+Reporting Interface (AERI) in JBoss Tools. You can read about configuring
+error reporting in JBoss Tools here: http://tools.jboss.org/usage/#Error%20reporting
```

Comment:
You can read about configuring error reporting in JBoss Tools link:/usage/#error-reporting[here].